draft-ietf-rats-msg-wrap.md

```diff
@@ -232,26 +232,26 @@ both Reference Values and Endorsements within the same `application/signed-corim
 shared by different conceptual messages.
 Future specifications may add new values to the `ind` field; see {{iana-ind-ext}}.
 
-## CMW CBOR Tags {#cbor-tag}
+## CMW Tagged Items {#cbor-tag}
 
-CMW of type CBOR Tag derive their tag numbers from a corresponding CoAP Content-Format ID using the `TN()` transform defined in {{Appendix B of RFC9277}}.
+CMW Tagged Items are CBOR Tags that derive their tag numbers from a corresponding CoAP Content-Format ID using the `TN()` transform defined in {{Appendix B of RFC9277}}.
```

Comment:
```suggestion
CMW Tags are CBOR Tags that derive their tag numbers from a corresponding CoAP Content-Format ID using the `TN()` transform defined in {{Appendix B of RFC9277}}.
```

Reply:
Isn't this re-introducing the problem of CMW Tag and CBOR Tag being confusing?

Reply:
> Isn't this re-introducing the problem of CMW Tag and CBOR Tag being confusing?

yes.

Quoting the [relevant bit from the Issue](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/issues/148#issuecomment-2652690390) where this is discussed:

> _"The phrase to name this should not end in "tag" to distinguish from "CMW tag" and the like."_

Reply:
As long "item" is not used in this context adding even more confusion, I am okay with this compromise.